dnsmasq.sls

```diff
@@ -24,8 +24,17 @@ dnsmasq:
 #
 # dnsmasq: failed to create listening socket for 172.17.0.1: Cannot assign requested address
 # dnsmasq: unknown interface docker0
-/lib/systemd/system/dnsmasq.service:
-  file.line:
-    - mode: ensure
-    - after: {{ '[Unit]' | regex_escape }}
-    - content: After=docker.service
+dnsmasq_systemd_override:
+  file.managed:
+    - name: '/etc/systemd/system/dnsmasq.service.d/after-docker.conf'
```

Comment:
There should be no need to quote the file path.

Reply:
removed the quotes